modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -7043,19 +7043,24 @@ type Flamingos2Dynamic {
   lyotWheel: Flamingos2LyotWheel!
 
   """
-  Flamingos 2 mask, if any.
+  Flamingos 2 FPU, if any.
   """
-  mask: Flamingos2Mask
+  fpu: Flamingos2FpuMask
```

Comment:
I switched the field name here to match GMOS.  GMOS has the same builtin vs. custom mask so it seemed to me that they should have the same name.  If `mask` is a better name, we could switch them both.